lib/web_ui/test/window_test.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.6
+// @dart = 2.12
```

Comment:
We're probably just importing a helper or a constant from `history_test.dart`. We could move that to another `history_helper.dart`.

But `matchers.dart` may require more work since it has even more dependencies.

Reply:
Done (migrated the helper test libs to null-safety)